src/plugins/installer.py

```diff
@@ -0,0 +1,92 @@
+import os
+import pathlib
+
+from helperFunctions.install import read_package_list_from_file, run_cmd_with_logging
+
+
+class AbstractPluginInstaller:
```

Comment:
should this maybe inherit from `abc.ABC` and use `@abstractmethod` decorators?

Reply:
I don't think so.
Lets take for example `install_docker_images`. If it would be an `@abstractmethod` every plugin installer would have to implement it. So plugins that don't need docker images would have to have an empty function.

Should we maybe rename `AbstractPluginInstaller`?